sdk/testing/build.gradle

```diff
@@ -9,6 +9,7 @@ ext.moduleName = 'io.opentelemetry.sdk.testing'
 dependencies {
     api project(':api:all')
     api project(':sdk:all')
+    api project(':sdk:metrics')
```

Comment:
If we have testing depend on metrics, we need to give it an alpha version number.

Reply:
Yeah - presumably we have many extensions we'll need to give alpha too (logging for example). Let me try addressing that more generically outside this PR.

Reply:
I don't think we have a requirement of testing artifact stability, do we? [not that I care too much if the testing artifact is versioned `alpha`, really]